MobiFlight/Joysticks/Octavi/Octavi.cs

```diff
@@ -121,9 +121,9 @@ private void InputReceiver_Received(object sender, System.EventArgs e)
             {
                 OctaviReport OReport = new OctaviReport();
                 OReport.parseReport(inputReportBuffer);
-                List<int> btnz = octaviHandler.toButton(OReport);
-                foreach (int i in btnz) {
-                    TriggerButtonPress(i);
+                List<(int, MobiFlightButton.InputEvent)> btnz = octaviHandler.toButton(OReport);
```

Comment:
```suggestion
                var buttons = octaviHandler.toButton(OReport);
```

We've been slowly moving towards `var` when possible. And I realize it wasn't your original code, but we can spell the whole word `buttons` out, there's no need to abbreviate it.